flow/designs/asap7/ibex/config.mk

```diff
@@ -3,7 +3,32 @@ export PLATFORM               = asap7
 export DESIGN_NICKNAME        = ibex
 export DESIGN_NAME            = ibex_core
 
-export VERILOG_FILES         = $(sort $(wildcard $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/*.v))
+export VERILOG_FILES = $(DESIGN_HOME)/src/$(DESIGN_NICKNAME)/ibex_pkg.sv \
```

Comment:
any reason not to use a glob pattern here and have more terse code?

Makefiles will never have anything but the crudest dependency notions by 2025 standards.

Reply:
DESIGN_DIR is terser

Reply:
Not really